lxd/instance/drivers/driver_lxc.go

```diff
@@ -6904,7 +6933,14 @@ func (d *lxc) FileSFTP() (*sftp.Client, error) {
 func (d *lxc) stopForkfile(force bool) {
 	// Make sure that when the function exits, no forkfile is running by acquiring the lock (which indicates
 	// that forkfile isn't running and holding the lock) and then releasing it.
-	defer func() { locking.Lock(context.TODO(), d.forkfileRunningLockName())() }()
+	defer func() {
```

Comment:
This is the only change I am not sure about. Is it ok to just return here in case the lock cannot be obtained? I know it's theoretical since `context.TODO()` is set.

Reply:
Yes that's fine.